src/menu/types.js

```diff
@@ -127,6 +127,10 @@ export type StatefulContainerPropsT = {
   getChildMenu?: (ref: {current: HTMLElement | null}) => ?{
     current: HTMLElement | null,
   },
+  /* Indicates where the highlighted item would show in a view.
+  If `auto`, we scroll the least amount possible to ensure that the highlighted item is visible.
+  If `center`, highlighted item would occur in the center of a view. */
+  initialScrollAllignInView?: 'auto' | 'center',
```

Comment:
What do you think about not exposing this setting for customization? 🤔

Reply:
yeah - let's just replace the default behavior